.changeset/kind-foxes-live.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nordeck/matrix-meetings-bot': patch
+---
+
+Update room layout when jitsi widget is added to an already existing room
```

Comment:
It's not about Jitsi but about the widget layout not being updated correctly when the widgets change—dependent on whatever is configured by the operator of the bot.

> Apply the configured room layout when widgets are added or removed.